content/en/docs/tasks/security/authorization/authz-td-migration/index.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Trust domain migration for authorization policy.
+description: Shows how to migrate from one trust domain to another without changing authorization policy.
+weight: 40
+keywords: [security,access-control,rbac,authorization,trust domain, migration]
+---
+
+This task shows you how to migrate from one trust domain to another without changing authorization policy.
+
+In Istio 1.4, we introduce an alpha feature to support {{< gloss >}}trust domain migration{{</ gloss >}} for authorization policy. This means if an
+ Istio mesh needs to change its {{< gloss >}}trust domain{{</ gloss >}}, the authorization policy doesn't need to be changed manually.
+ In Istio, if a {{< gloss >}}workload{{</ gloss >}} is running in namespace `foo` with the service account `bar`, and the trust domain of the system is `my-td`,
+ the identity of said workload is `spiffe://my-td/ns/foo/sa/bar`. By default, the Istio mesh trust domain is `cluster.local`,
+ unless you specify it during the installation.
+
+## Before you begin
+
+The activities in this task assume that you:
+
+1. Read the [authorization concept](/docs/concepts/security/#authorization).
+
+1. Create a trust domain profile.
```

Comment:
The step and the following one would be better combined.
```
1. Install Istio with mesh trust domain set to `old-ts`:

    {{< text bash >}}
    $ echo "apiVersion: install.istio.io/v1alpha2
      kind: IstioControlPlane
    ...
    $ istioctl manifest apply --set profile=demo -f td-installation.yaml
    {{< /text >}}
```

Reply:
Also it would be good to use a consistent style with other tasks that start Istio with custom settings (e.g., cat vs. echo). This one looks like a good style to follow: https://preliminary.istio.io/docs/tasks/security/dns-cert/#before-you-begin

Reply:
Done. Thanks!